tests/bluetooth/host/keys/testing_common_defs.h

```diff
@@ -17,3 +17,15 @@
 #define BT_ADDR_LE_3 ((bt_addr_le_t[]) { { 0, { { 0x0C, 0x89, 0x67, 0x45, 0x23, 0x01 } } } })
 #define BT_ADDR_LE_4 ((bt_addr_le_t[]) { { 0, { { 0x0D, 0x89, 0x67, 0x45, 0x23, 0x01 } } } })
 #define BT_ADDR_LE_5 ((bt_addr_le_t[]) { { 0, { { 0x0E, 0x89, 0x67, 0x45, 0x23, 0x01 } } } })
+
+bool all_startup_checks_executed;
```

Comment:
This is incorrect. You are effectively declaring this variable in every file this header is included.